test/sync/index.js

```diff
@@ -64,12 +64,14 @@ describe('Data Sync', function() {
           app,
           agSync: {
             OfflineClient,
-            CordovaNetworkStatus,
-            CacheOperation,
-            getUpdateFunction
+            CordovaNetworkStatus
           },
           gql,
-          ToggleNetworkStatus
+          ToggleNetworkStatus,
+          offixCache: {
+            CacheOperation,
+            getUpdateFunction
```

Comment:
I've noticed that neither the documentation nor the ionic-showcase uses the `getUpdateFunction` so for what are we exactly using it? (Isn't better to test the default configuration for starting)

Reply:
It is inspired by ionic-showcase: https://github.com/aerogear/ionic-showcase/blob/18bfc75400125644df3db6b3ac0eb91ca44d66fe/src/app/services/sync/cache.updates.ts#L15-L17

Reply:
Because it seems to me that they are automatically created: https://github.com/aerogear/offix/blob/333fad209ad24e1a51dd1b39b28865a70e3af3a0/packages/offix-cache/src/cache/mutations.ts#L44, maybe it is something added in the latest versions